### DISCUSSION TITLE
Possibility to add "Trackers to avoid"

### DISCUSSION THREAD

**Discussion by Vyerni** [cross-seed/cross-seed#66] [Ideas]
Hi, 

First, apologies if this is posted in the wrong place, rather than issues. 

But given that in the config file, there's an option to add trackers to search, I was wondering if it could be implemented that it'll search all torrents in your Jackett, but have an exclusion on a couple of particulars?

**Comment by mmgoodnow** [comment]
That's not a bad idea! Feel free to open an issue for it (I have some other things to get to first). I also welcome pull requests :)

**Reply by zakkarry** [reply]
Can we consider this closed now that specific Torznab URL's are used for prowlarr/jackett/Torznab?
